src/__tests__/plugin-test.js

```diff
@@ -0,0 +1,43 @@
+import Microcosm  from '../Microcosm'
+
+describe('Microcosm', function() {
+
+  it ('can add plugins', function(done) {
+    let m = new Microcosm()
+
+    m.addPlugin({
+      register: function(app, options, next) {
+        next()
+      }
+    })
```

Comment:
Worth noting. These tests receive the same processing as the main source code. They all pass through the same webpack configuration.